### ISSUE TITLE
Fix for @apply reports double changes

### ISSUE THREAD

**Issue by stramel** [Polymer/polymer-linter#144]
I assume this is intended but is a bit confusing. 

```sh
  Made 12 changes to `mtz-widget.html`

Fixed 6 warnings.
```

these are all `@apply(--mtz-mixin);` to `@apply --mtz-mixin;` but it reports double (1 for each paren removed)

@rictic

**Comment by rictic** [comment]
This is intended. It's a bit confusing in the common case, but it ensures that we do give a clear reporting of where changes happen. A fix is made up of changes, and those changes can in theory take place across multiple files. So you might apply a fix of a warning in `a.html` but that might involve changes to `b.html` and `c.html`.

My intuition is that what you'd want to see is the files that were changed, and the count of warnings that were fixed. Maybe we should drop the number of changes per file though, as that's kinda useless.

WDYT of:

```
Made changes to:
    mtz-widget.html

Fixed 6 warnings.
```

**Comment by stramel** [comment]
Yeah, I could see maybe a breakdown? File and number of warnings fixed? I think you have done a good job for an initial release but think it could be cleared up a bit still. 👍

**Comment by rictic** [comment]
Printing the number of warnings could end up with a similar issue though, as you'd have to either double-count or, in some cases, report that we changed a file but fixed no warnings in that file. This is hypothetical for the moment because we don't yet have any warnings that require touching multiple files to fix, but imagine that to fix a warning in `a.html` but to do that it changes `b.html`. How do we report that?

```
    Fixed 1 warning in a.html
    Fixed 1 warning in b.html

Fixed 1 warning.
```

That doesn't seem right.

Or:

```
  Fixed 1 warning in a.html
  Fixed 0 warnings in b.html (though it was changed)

Fixed 1 warning.
```

I think the simplest result that's still clear is listing the files changed and then the number of warnings total fixed.

**Comment by usergenic** [comment]
Triaging... Is there a todo here or should this be marked won't fix...?

**Comment by rictic** [comment]
I've got a fix locally, should be out shortly.
